CMakeLists.txt

```diff
@@ -529,6 +529,15 @@ if (ISLE_BUILD_APP)
       ISLE/3ds/config.cpp
     )
   endif()
+  if(Python3_FOUND)
+    execute_process(
+      COMMAND ${Python3_EXECUTABLE} tools/png2h.py ISLE/res/arrow.png ISLE/res/busy.png ISLE/res/no.png
+      WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+      RESULT_VARIABLE PNG2H_RESULT
+      OUTPUT_QUIET
+      ERROR_QUIET
+    )
```

Comment:
This needs to be `add_custom_command`, the input and output files should be explicitly enumerated, and the generated headers should be part of `add_executable(isle)`.
Otherwise, the headers are always generated, causing `isle` to rebuild on every `cmake` invocation.

Reply:
Also not sure this should be part of CMake because `PIL` is not part of the Python standard library.

Reply:
Oh, didn't know that's possible. I'll fix it then